Tests/LegacyUnit/BagBuilder/AbstractTest.php

```diff
@@ -520,6 +520,73 @@ public function testWhereClauseCanSelectPids()
         );
     }
 
+    public function testWhereClausePartGetKeyMustNotBeEmpty()
+    {
+        $this->expectException(
+            \InvalidArgumentException::class
+        );
+        $this->expectExceptionMessage(
+            'The parameter $key must not be empty.'
+        );
+
+        $this->subject->getWhereClausePart('');
+    }
+
+    public function testWhereClausePartSetKeyMustNotBeEmpty()
```

Comment:
For new tests, could you please use the `@test` annotation instead of the `test` prefix? This will help keep the test names a bit shorter.